torch/csrc/jit/codegen/cuda/executor_utils.cpp

```diff
@@ -728,84 +743,63 @@ void validateVectorizedSplits(
   }
 }
 
-} // namespace
-
-void validateVectorizedTensors(
-    kir::Kernel* kernel,
-    const at::ArrayRef<IValue>& inputs,
-    const std::vector<at::Tensor>& outputs,
-    caching::ExecutorCompileTimeInfoCache* data_cache,
-    kir::ExpressionEvaluator& expr_eval) {
-  FUSER_PERF_SCOPE("FusionExecutor::validateVectorizedTensors");
-
-  validateAlignedVectorizedTensors(
-      kernel, inputs, outputs, data_cache, expr_eval);
-
-  validateMisalignedVectorizedTensors(
-      kernel, inputs, outputs, data_cache, expr_eval);
-
-  validateVectorizedSplits(kernel, expr_eval);
-}
-
-kir::ExpressionEvaluator bindKernelInputs(
-    const at::ArrayRef<IValue>& aten_inputs,
-    kir::Kernel* kernel,
-    bool check_consistency) {
-  FUSER_PERF_SCOPE("executor_utils::BindKernelInputs");
-
-  TORCH_INTERNAL_ASSERT(
-      kernel->inputs().size() == aten_inputs.size(),
-      "Something went wrong configuring launch. Inputs no longer match.");
-
-  kir::ExpressionEvaluator expr_eval;
-  const auto& inputs = kernel->inputs();
-
-  for (const auto i : c10::irange(inputs.size())) {
-    const auto input = inputs[i];
-
-    if (auto tensor_input = dynamic_cast<TensorView*>(input)) {
+template <typename EXPR_EVALUATOR>
```

Comment:
Could you put these functions back in the original order for now as it's hard to parse the mechanical vs non-mechanical changes.

Reply:
Moved `validateVectorizedTensors` into its original spot.

It still looks a little messy, since I merged a chunk of code shared by `bindFusionInputs` and `bindKernelInputs` into `bindInputForExprEvaluation`

Reply:
This helped a ton, thanks.